tests/test_echo.py

```diff
@@ -0,0 +1,403 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test the embedded echo server."""
+
+import unittest
+import json
+import os
+import unittest
+
+from testbench import rest_server
+
+
+class TestEcho(unittest.TestCase):
+    def setUp(self):
+        self.client = rest_server.server.test_client()
+
+    def test_delete(self):
+        expected_args = {"arg1": "value1", "arg2": "value2"}
```

Comment:
Maybe worth factoring out some of this duplicated data across tests? Up to you either way, though

Reply:
I am Okay with the dups, thanks.